bramkouwen/Neatly-In-Motion
# Adding Images to Website

## What does this PR do?
* It adds a img folder
* It places photos for the website in the img folder

## How do you check this PR?
* Pull down the branch
* Check the added img folder 
* Check the added images

## Screenshots 
None.
